Cargo.toml

```diff
@@ -53,6 +53,7 @@ name = "c32_bench"
 harness = false
 
 [dependencies]
+pico-args = "0.3.1"
```

Comment:
Can we please not add new dependencies?  Thanks

Reply:
That doesn't mean we should put it into the stacks library.  Honestly, we should be _removing_ dependencies, not adding to them.

Reply:
Thanks!